### DISCUSSION TITLE
msw-storybook-addon 설치 건의

### DISCUSSION THREAD

**Discussion by WaiNaat** [woowacourse-teams/2023-pium#28] [뜨거운 토론]
안녕하세요.

### 안건

msw-storybook-addon 설치해도 되나요?
[패키지 바로가기](https://storybook.js.org/addons/msw-storybook-addon)

### 설명

스토리북에서 msw handlers를 부드럽게 사용할 수 있습니다.
저희가 어떤 방식으로 세부 컴포넌트를 구현할 지는 모르겠지만 만약 그 안에서 api 통신을 진행하는데 스토리북으로 컴포넌트를 보고 싶을 경우에는 msw를 켜야 하는데요.

![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/cc93cc0b-4f59-48da-b2b1-fbc5eade6f6f)

애드온을 설치하면 parameters에 한 줄 추가해서 쉽게 가능합니다.
애드온 없이 하려면 보일러플레이트가 이거보다 더 많이 발생합니다.

현재 스토리북용 경량화 핸들러도 같이 만들어놓았기 때문에 적용에 오래 걸리진 않아요

**Comment by bassyu** [comment]
LGTM!

**Comment by WaiNaat** [comment]
원만한 합의하에 설치하였습니다~~
